protobuf-converter/src/main/java/io/confluent/connect/protobuf/ProtobufData.java

```diff
@@ -296,7 +300,9 @@ private Object fromConnectData(
                 if (fieldDescriptor == null) {
                   throw new DataException("Cannot find field with name " + fieldName);
                 }
-                messageBuilder.setField(fieldDescriptor, fieldValue);
+                if (fieldValue != null) {
```

Comment:
There is already a null check on L289. I assume `fieldValue` can be mutated in between so we need another check?

Reply:
Yes there is a mutation on L295